koocook_core/urls/__init__.py

```diff
@@ -16,7 +16,13 @@
     path('comments/<int:item_id>/comment', comment_handler.handle, name='comment-self', kwargs={"alias": 'comment'}),
     path('comments/<int:pk>/rate', comment_handler.handle, name='comment-rate', kwargs={"alias": 'rate'}),
     path('profile/', include('koocook_core.urls.profile'), name='profile'),
+    path('cookware/', views.RecipeEquipmentView.as_view(), name='equipment-all'),
+    path('cookware/<int:pk>', views.RecipeEquipmentDetailView.as_view(), name='equipment'),
+    path('ingredients/', views.RecipeIngredientsView.as_view(), name='ingredients'),
+    path('ingredients/<int:pk>', views.IngredientDetailView.as_view(), name='ingredient'),
+    path('recipes/', include('koocook_core.urls.recipes'), name='recipes'),
     path('recipes/', views.PreferredRecipeStreamView.as_view(), name='recipe-all'),
+    path('recipes/unit_conv', recipe_handler.handle, name='recipe-unit-conv', kwargs={"alias": 'unit_conv'}),
     path('recipes/<int:recipe_id>', views.handle_recipe, name='recipe'),
     path('recipes/<int:item_id>/comments', recipe_handler.handle, name='recipe-comments', kwargs={"alias": 'comment'}),
     path('recipes/<int:pk>/edit', views.RecipeUpdateView.as_view(), name='recipe-edit'),
```

Comment:
Why the redundancy?

Reply:
Legacy code. Removal pending

Reply:
Fixed in cf75dd5f99e1a8d0c1eced6fb56c6d1ef9a38542 for you